apps/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../tsconfig.json",
   "include": [
     "./*/src",
-    "./libs/*/src"
+    "./libs/*/src",
+    "./*/__sanity__"
```

Comment:
```suggestion
```
Is there a reason why we're including `__sanity__` tests in the production build?